ciao-launcher/qemu.go

```diff
@@ -358,7 +358,8 @@ func launchQemuWithNC(params []string, fds []*os.File, ipAddress string) (int, e
 		ncString := "socket,port=%d,host=%s,server,id=gnc0,server,nowait"
 		params[len(params)-1] = fmt.Sprintf(ncString, port, ipAddress)
 		var errStr string
-		errStr, err = qemu.LaunchQemu(context.Background(), params, fds, qmpGlogLogger{})
+
+		errStr, err = qemu.LaunchCustomQemu(context.Background(), "", params, fds, qmpGlogLogger{})
```

Comment:
@mcastelino I changed that so that each device can bring its own set of fds. Config.FDs is gone and each device QemuParams() call should append its fds to the overall fd slice.
See commit #c1a5e0177bb04392e737c7dc757b7a3f3423cead.